operations/Makefile

```diff
@@ -4,10 +4,15 @@ SHELL := bash
 TF_ENV = "dev" #VAR Your target environment (e.g. dev, test, staging, prod)
 TF_STAGE = ""  # not surfaced
 
+AZ_RESOURCE_PREFIX = "pdh$(TF_ENV)" ## VAR Azure environment resource prefix (e.g. pdhtest)
+
+ZDD_SOURCE_SLOT = "candidate" #VAR Source function app slot for ZDD deployment
+ZDD_TARGET_SLOT = "production" #VAR Destination function app slot to deploy to
```

Comment:
These shouldn't need to be changed, but providing the option to override in the future.